test/acceptance/user.controller.acceptance.ts

```diff
@@ -130,6 +133,74 @@ describe('UserController', () => {
     });
   });
 
+  describe('authentication functions', () => {
+    it('login returns a valid token', async () => {
+      const newUser = await userRepo.create(user);
+      await client
+        .post('/users/login')
+        .send({email: newUser.email, password: newUser.password})
+        .expect(200)
+        .then(getToken);
+
+      function getToken(res: Response) {
+        // is it good to return the token as text?
+        const token = res.text;
+        expect(token).to.not.be.empty();
+      }
+    });
+
+    it('login returns an error when invalid credentials are used', async () => {
+      const newUser = await userRepo.create(user);
+      newUser.password = 'wrong password';
+      await client
+        .post('/users/login')
+        .send({email: newUser.email, password: newUser.password})
+        .expect(401);
+    });
+
+    it('/me returns the current user', async () => {
+      const newUser = await userRepo.create(user);
+      let token = '';
+      await client
+        .post('/users/login')
+        .send({email: newUser.email, password: newUser.password})
+        .expect(200)
+        .then(getToken);
+
+      function getToken(res: Response) {
+        // is it good to return the token as text?
+        token = res.text;
+      }
+      // MongoDB returns an id object we need to convert to string
+      // since the REST API returns a string for the id property.
+      newUser.id = newUser.id.toString();
```

Comment:
Have you considered using `toJSON` helper from `@loopback/testlab` instead? See https://github.com/strongloop/loopback-next/tree/master/packages/testlab#toJSON

```
const me = _.pick(toJSON(newUser), ['id', 'email']));
```

Reply:
This comment does not seem to be addressed, PTAL.

(A detail not preventing this PR from being approved.)

Reply:
@bajtos Hmm we already switched to `toJSON`, could you elaborate more about your expected implementation? thanks.

Reply:
Can we remove this line now? I believe that `toJSON` will take care of converting `id` from ObjectID to a string.